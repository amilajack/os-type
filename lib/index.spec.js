const addon = require('.');

describe('basic', () => {
    it('should return object', () => {
        expect(addon.currentPlatform()).toMatchSnapshot();
    });
});