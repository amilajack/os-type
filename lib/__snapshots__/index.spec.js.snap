// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic should return object 1`] = `
Object {
  "osType": "unknown",
  "version": "0.0.0",
}
`;
